ui/src/components/about/AboutUsContainer.js

```diff
@@ -11,6 +11,7 @@ import StaffSection from './StaffSection';
 import StorySection from './StorySection';
```

Comment:
[Comment] dont forget to name the branch with the issue number prefix next time. "494-about-mission".

overall good job. aside from a nitpick, everything looks good